integrations/hello-world-ng11-ivy/bundlesize.config.json

```diff
@@ -3,7 +3,7 @@
     {
       "path": "./dist-integration/main.*.js",
       "target": "es2015",
-      "maxSize": "250.12 kB",
+      "maxSize": "250.58 kB",
```

Comment:
Caretaker note: it grew up because we merged fixes related to router, websocket and devtools plugins.